backend/upload.js

```diff
@@ -58,6 +58,16 @@ export function deleteObject (s3, bucket, key) {
   return s3.deleteObjects(params).promise();
 };
 
+export function deleteObjects (s3, bucket, keys) {
```

Comment:
I think we can use the above method for this purpose by adding an if condition to check if the parameter `key` is an Array or a single element.

Reply:
thanks, I have changed that function, now it checks the type of key passed to to it for code reuseability.